apps/modernization-ui/src/apps/page-builder/components/CreateCondition/CreateCondition.tsx

```diff
@@ -37,8 +41,8 @@ export const CreateCondition = () => {
         }).then((response: any) => {
             const data = response || [];
             const codingSystemList: never[] = [];
-            data.map((each: { value: never }) => {
-                codingSystemList.push(each.value);
+            data.map((each: { conceptCode: never }) => {
+                codingSystemList.push(each.conceptCode);
             });
             setSystemOptions(codingSystemList);
```

Comment:
We should avoid using `:any` and `never[]`. The proper types can be seen in the generated API calls. I think this is also causing the drop down selections to be empty:
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/78672a8b-5b8a-49ae-afdb-7c4c84590cf8)



I think this fixes it for the `Coding System`

```typescript
const fetchCodingSystemOptions = () => {
    ValueSetControllerService.findConceptsByCodeSetNameUsingGet({
        authorization: `Bearer ${state.getToken()}`,
        codeSetNm: 'CODE_SYSTEM'
    }).then((response: Concept[]) => {
        setSystemOptions(response);
    });
};
```

```typescript
<select
    className="field-space"
    name="codingSystem"
    value={system}
    onChange={(e: any) => setSystem(e.target.value)}>
    <option>- Select -</option>
    {systemOptions.map((o, i) => (
        <option key={i} value={o.conceptCode}>
            {o.display}
        </option>
    ))}
</select>
```

Reply:
Done

Reply:
Would you mind fixing the other drop down values that list `never` as their type? 

1. `fetchProgramAreaOptions`
2. `fetchGroupOptions`
3. `fetchProgramAreaOptions`